.eslintrc

```diff
@@ -9,5 +9,26 @@
   ],
   "rules": {
     "prettier/prettier": "error"
-  }
+  },
+  "overrides": [
+    {
+        "files": ["**/*.md"],
```

Comment:
Indentation seems off in this file

Reply:
Yeah, reformatted. Might make sense to just convert this to the JS version later